content/pl/docs/preparation/macOS/_index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: MacOS
+linkTitle: MacOS
+weight: 1
+description: >
+    Opis przygotowania nośnika instalacyjnego z MacOS.
+---
+
+
+# Pobieranie instalatora z gibMacOS
```

Comment:
gibMacOS działa również na innych systemach, jako że to jest skrypt Pythonowy. Zmienia się tylko sposób przygotowania instalacji poprzez ten tool.

Reply:
zgadza sie ale nie rozumiem sensu tego commenta